mxd/postman/MXD_Management_API.json

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
`management_api_tests.json` would be a better name for this file.

There are two variables `MANAGEMENT_URL` and `PROVIDER_PROTOCOL_URL` that are undefined in the collection, so running this via `newman` won't work. You probably have those variables in your local postman installation, which it why it works - locally.

Again: have you tested this locally **with newman**, or in a CI run?

Reply:
@paullatzelsperger  I've removed the unused variables, namely MANAGEMENT_URL and PROVIDER_PROTOCOL_URL, as they are not being utilized within the collections.

yes these are tested using newman CI.

![image](https://github.com/eclipse-tractusx/tutorial-resources/assets/142776138/f82194e7-5b8e-40d5-a2de-06c3a18cfc42)